### ISSUE TITLE
Add Ethereum name or address to package.json

### ISSUE THREAD

**Issue by lastmjs** [trufflesuite/truffle#2330]
I would like to see an Ethereum name or address added to the `ethereum` field of the package.json file. This will allow consumers of the project to know a cannonical address to send funds to, for example donations.

I'm building an application that will allow developers to automatically donate ETH to all of their npm dependencies that have Ethereum addresses in their package.json files. I'm not quite ready to share the project, but I'd love to see an Ethereum address in this project's package.json file.

**Comment by eggplantzzz** [comment]
The idea is that I don't think we should be sticking arbitrary fields there. That can't be a good practice. If it is something that is standardized I have much less of a problem with it. It really has nothing to do with ease. I'll take a look at your docs though.

**Comment by lastmjs** [comment]
Yes, I'm starting to see and agree. So two options seem must appealing now. One is to enable GitHub Sponsors and include an Ethereum address or name in the FUNDING.yml file. This is how it would look and I have tested it:

```yml
custom: ['ethereum/truffle.eth']
```

You could replace truffle.eth with the actual address if you wanted to. Also, a real URL might be better, but not necessary. What I have above seems the most project agnostic.

Another option that I'm working on getting standardized is essentially a field in the package.json under something like support -> backing -> donations. It's still nebulous but there is work being done as linked to above

**Comment by eggplantzzz** [comment]
I think we like `FUNDING.yml` idea! We are currently discussing what are strategy is going to be for taking donations. Thanks for the proposal!

**Comment by lastmjs** [comment]
Something to be aware of: https://blog.npmjs.org/post/187382017885/supporting-open-source-maintainers

I'm pretty much waiting to see what npm is going to do before making any more plans with Sustainus.

That being said, I think if GitHub added native support for Bitcoin and Ethereum and potentially other cryptocurrencies into the FUNDING.yml schema, that could be a win for crypto. Show your support here: https://github.com/lastmjs/enable-cryptocurrency-in-funding-yml

Also, anyone interested in making this a reality, please email support@github.com and ask for cryptocurrencies in FUNDING.yml. I was told by a member of the GitHub Sponsors team that it is likely they will support it if enough people ask
